src/BizHawk.Client.Common/savestates/ZipStateLoader.cs

```diff
@@ -9,7 +9,8 @@ namespace BizHawk.Client.Common
 	public class ZipStateLoader : IDisposable
 	{
 		private ZipArchive _zip;
-		private Version _ver;
+		private Version _zipVer;
```

Comment:
Could we just bump the state version to something new instead of introducing a second version?

Reply:
Movies do not have this included it seems (seems like this was meant only for savestates?)

Reply:
Ended up just making all movies put in the BizState 1.0 file and bumped it up to 1.0.2.

On related https://github.com/TASEmulators/BizHawk/pull/3345#discussion_r936672311